README.md

```diff
@@ -16,6 +16,8 @@ The Open Neural Network Exchange implementation in MLIR (http://onnx.ai/onnx-mli
 
 The preferred approach to using and developing ONNX-MLIR is to used Docker Images and Containers, as getting the proper code dependences may be tricky on some systems. Our instructions on using ONNX-MLIR with dockers are [here](docs/Docker.md).
 
+If you intend to develop code, you should look at our [workflow](docs/Workflow.md) document which help you setup your docker environment in a way that let you contribute code easily.
```

Comment:
docker environment -> environment (some ppl may not want to use docker in their working environment).

Reply:
Right now we do use Docker in many places, and we have Dockerfile, so we are really tied to that. I am told that there are ways to execute Docker images on other systems than DockerDesktop. So I feel it is ok to use docker here.